src/plugins/featureFlags.ts

```diff
@@ -5,9 +5,10 @@ declare const window: any
 
 /**
  * Default flag values when LD is not available.
- * Uses "business-edit" project (per LD client id in config).
+ * Uses "namerequest" project (per LD client id in config).
  */
 const defaultFlagSet: LDFlagSet = {
+  'enable-sp-gp-dba': false, // by default, don't show firm Register button
```

Comment:
I added this FF in LaunchDarkly. It is the same flag as in sbc-auth to allow only SP/GP registrations from a NR (else goes to OneStop).

Currently, it is True in Dev and Test, and False in Prod.

When SP/GP is released to the public, this flag will have to be enabled in Prod (for both sbc-auth and namerequest LD projects).